files/root/grafana_tools/backup_and_sync/config

```diff
@@ -0,0 +1,7 @@
+destination_host=
```

Comment:
Suggest: add a comment that this file is managed by Puppet (local changes will be lost).

Reply:
Again, these files come from another git repo. I don't want to start editing or recreating the files.
But a great idea for a future improvement.

Reply:
thumbs up